scripts/deploy/command.mjs

```diff
@@ -4,19 +4,45 @@ import process from 'node:process';
 import { workspacePackages } from '../common/utils.mjs';
 import { build } from '../publish/build.mjs';
 import { logAsSection } from '../publish/utils.mjs';
-import { deployProjectsToVercel } from './utils.mjs';
+import { deployProjectsToVercel, getVercelProjectId } from './utils.mjs';
+import { ENABLE_PREVIEW_DEPLOY } from './config.mjs';
+
 
 const EXCLUDED_PACKAGES = ['@rango-dev/widget-iframe'];
 
 // TODO: Working directory should be empty.
 async function run() {
+  /*
+    Deploys packages based on the state of the `ENABLE_PREVIEW_DEPLOY` environment variable.
+    if ENABLE_PREVIEW_DEPLOY is true, only packages that has project id in workflow environments will be deployed.
+    else private packages will be deployed.
+ */
+
   // Detect last release and what packages has changed since then.
   const packages = await workspacePackages();
   const privatePackages = packages.filter((pkg) => {
```

Comment:
I think you can make this a separate function (e.g. `getClientsListToBeDeployed`) and rename the variable to `listPackagesToBeDeployed`

Reply:
Done